apps/web/app/(app)/environments/[environmentId]/integrations/notion/components/ManageIntegration.tsx

```
@@ -70,74 +69,52 @@ export const ManageIntegration = ({
-            </TooltipTrigger>
-            <TooltipContent>{t("environments.integrations.notion.update_connection_tooltip")}</TooltipContent>
-          </Tooltip>
-        </TooltipProvider>
-        <Button
-          onClick={() => {
+    <>
+      <IntegrationListPanel
+        environment={environment}
+        statusNode={
+          <>
+            <span className="mr-4 h-4 w-4 rounded-full bg-green-600"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)